### ISSUE TITLE
Errors catching

### ISSUE THREAD

**Issue by mvasin** [jakubboucek/schedule-reception-doctors#3]
Failover is not created. When something fail (most often external data source), app breaks without user notification.

We need some proper error messages on screen for sure. It's a public place, we need to fail gracefully.

**Comment by rajpatel507** [comment]
@mvasin @jakubboucek Can I take this one?

**Comment by jakubboucek** [comment]
You can.

**Comment by rajpatel507** [comment]
Thanks.

**Comment by jakubboucek** [comment]
@mvasin are you require running app from local storage?

Explain: Exists two major variant how you car run app: 
1) save it directly to each display and run it locally via file:// protocol (like a `file:///c:\Users\Display\Documents\app\index.html`) or
2) open as website from internet via HTTP ([like this app](https://s3-eu-west-1.amazonaws.com/schedule-reception-doctors/master/index.html)).

First can simple make failover when display will by offline ([already finished](https://github.com/jakubboucek/schedule-reception-doctors/commit/c1dee41b16c62896bd47d52e9e12684b7edf19f1#diff-c1e74ceb5b235b9b4cf7fb244d8479e0R5) by @rajpatel507).
Second variant can be simply upgraded anytime, but offline failover [must be solver through Service Worker](https://css-tricks.com/serviceworker-for-offline/).

**Comment by mvasin** [comment]
The schedule we are making will be integrated into Wordpress site, as a page. And the main use case - opening that page via TV's in-built browser. So - second variant.

**Comment by 7flash** [comment]
We can show error message when API is unavailable. It is enough.

**Comment by mvasin** [comment]
Yes, I think it's sufficient to show error messages on unsuccessful data fetch.

What can make a data fetch unsuccessful?

- There could be **HTTP access errors**, in this case we can **show 'Сервер недоступен' ('server unavailabe') message**.
- **JSON** could be **invalid** as well. Theoretically, information provider can change data structure any time, we have no agreement with them. It would be nice to check that at least it's not empty: there is at least one doctor entry with at least one schedule entry, according to the current data structure we currently expect from them. Otherwise there will be nothing to show, and we can display **'Данные отсутствуют' ('no data') message**.

@rajpatel507 Rajesh, will you nail this?

**Comment by jakubboucek** [comment]
FYI: JSON from API should be valid JSON, but can cause unexpected issue. Currently sent weird data in `scheludes` object: some items is empty and [I must fixed it](https://github.com/jakubboucek/schedule-reception-doctors/blob/master/source/app.js#L187).

Can you pls catch all errors by error event? Example:
```javascript
window.addEventListener('error', function (e) {
    var error = e.error;
    console.log(error);
});
```

I imagine handler like a:
```javascript
   $("#content").hide();
   $("#error").show().text(…);
   window.setTimeout(window.location.reload, 60000);
```

**Comment by rajpatel507** [comment]
I will work on this.
